src/plugins/auto_batch/src/compiled_model.cpp

```diff
@@ -154,93 +129,103 @@ std::pair<CompiledModel::WorkerInferRequest&, int> CompiledModel::GetWorkerInfer
-    return std::make_shared<AsyncInferRequest>(std::static_pointer_cast<SyncInferRequest>(syncRequestImpl),
-                                               inferRequestWithoutBatch,
-                                               _callbackExecutor);
+
+    auto sync_res = create_sync_infer_request();
+
+    ov::SoPtr<ov::IAsyncInferRequest> infer_request_without_batch = {
+        m_compiled_model_without_batch->create_infer_request(),
+        m_compiled_model_without_batch._so};
+    return std::make_shared<ov::autobatch_plugin::AsyncInferRequest>(
+        std::dynamic_pointer_cast<ov::autobatch_plugin::SyncInferRequest>(sync_res),
+        infer_request_without_batch,
+        get_callback_executor());
 }
 
-std::shared_ptr<ngraph::Function> CompiledModel::GetExecGraphInfo() {
-    return m_model_with_batch && m_model_with_batch->GetExecGraphInfo() ? m_model_with_batch->GetExecGraphInfo()
-                                                                        : m_model_without_batch->GetExecGraphInfo();
+std::shared_ptr<const ov::Model> CompiledModel::get_runtime_model() const {
+    return m_compiled_model_with_batch ? m_compiled_model_with_batch->get_runtime_model()
+                                       : m_compiled_model_without_batch->get_runtime_model();
 }
 
-void CompiledModel::SetConfig(const std::map<std::string, InferenceEngine::Parameter>& user_config) {
-    auto timeout = user_config.find(CONFIG_KEY(AUTO_BATCH_TIMEOUT));
-    if (timeout == user_config.end() || user_config.size() > 1) {
-        IE_THROW() << "The only config that can be changed on the fly for the AutoBatching the is the "
-                   << CONFIG_KEY(AUTO_BATCH_TIMEOUT);
+void CompiledModel::set_property(const ov::AnyMap& properties) {
+    auto time_out = properties.find(ov::auto_batch_timeout.name());
+    if (time_out == properties.end() || properties.size() > 1) {
+        OPENVINO_THROW("The only config that can be changed on the fly for the AutoBatching is the ",
+                       ov::auto_batch_timeout.name());
     } else {
-        m_timeout = ParseTimeoutValue(timeout->second.as<std::string>());
+        m_time_out = time_out->second.as<std::uint32_t>();
     }
 }
 
-InferenceEngine::Parameter CompiledModel::GetConfig(const std::string& name) const {
+ov::Any CompiledModel::get_property(const std::string& name) const {
     auto it = m_config.find(name);
     if (it != m_config.end()) {
         return it->second;
     } else {
         // find config key among networks config keys
-        auto param = m_model_without_batch->GetMetric(METRIC_KEY(SUPPORTED_CONFIG_KEYS));
-        for (auto&& configKey : param.as<std::vector<std::string>>()) {
-            if (configKey == name) {
-                return m_model_without_batch->GetConfig(configKey);
+        auto modelSupportedProperties = m_compiled_model_without_batch->get_property(ov::supported_properties.name());
```

Comment:
@zhaixuejun1993 As we synced,` supported_properties` should not be got from underlying plugin, please submit a separated PR to solve this issue.

Reply:
why get from underlying plugin, supported properties should be batch supported, for underlying plugin ,we can leverage ov::device::properties,

Reply:
ok. will be handled in the next PR